evm/src/executor/inspector/cheatcodes/expect.rs

```diff
@@ -259,6 +259,24 @@ pub fn apply<DB: Database>(
             Ok(Bytes::new())
         }
         HEVMCalls::MockCall0(inner) => {
+            // TODO: Does this increase gas usage?
+            data.subroutine.load_account(inner.0, data.db);
+
+            // Etches a single byte onto the account if it is empty to circumvent the `extcodesize`
+            // check Solidity might perform.
+            if data
+                .subroutine
+                .account(inner.0)
+                .info
+                .code
+                .as_ref()
+                .map(|code| code.is_empty())
+                .unwrap_or(true)
```

Comment:
there's also map_or https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.map_or

but I still prefer unwrap_or just because this is more readable